contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -137,7 +136,7 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
    * @notice Returns the proving key hash key associated with this public key
    * @param publicKey the key to return the hash of
    */
-  function hashOfKey(uint256[2] memory publicKey) public pure returns (bytes32) {
+  function hashOfKey(uint256[2] calldata publicKey) public pure returns (bytes32) {
```

Comment:
I don't know why this function is even `public` in the first place. This might break if an external contract with `memory` publicKey is calling this function.

Reply:
Don't know either. I can turn it into `internal` if we're sure nothing will break? cc @jinhoonbang